src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesReducer = createSlice({
+  name: 'categories',
+  initialState: [],
+  reducers: {
+    checkStatus() {
+      return 'Under construction';
+    },
```

Comment:
- According to [this requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md#:~:text=Includes%20a%20reducer%20that%20checks%20the%20status%20and%20always%20returns%20%22Under%20construction%22%20(the%20initial%20state%20should%20check%20to%20that%20string)), Kindly make sure the initial state check to the string Under Construction. it should be looking like this 
`checkStatus: (state, action) => {
      state =
        action.payload === "Under construction"
          ? "Under construction"
          : state
    },
`

Reply:
Hello @GedeonTS 

Thank you for your kind feedback. I shall make the changes and ask for a review again.

Regards